Game.java

```diff
@@ -64,29 +59,15 @@ public void assignPlayers() {
 
     public void playGame(Board board) {
         assignPlayers();
-        print(gameMode.toUpperCase() + " game is starting..");
-        print("Where would you like to go? (1, 2, 3, 4, 5, 6, 7, 8, 9)");
-        while (!board.checkForDraw() || !board.getState().equals("Game Over")) {
+        consoleGame.drawNewGame();
+        while (!board.isGameOver()){
```

Comment:
you pass in an argument of board to playGame which supercedes the global board, and then you re-assign board to the result of playerMakesMove, but you don't pass it to playerMakesMove, rather you use the board global variable.
This is trouble. What if your argument changed to gameBoard and all of the references to board in playeGame changed too. Then in playerMakesMove() you would have the wrong board.

This is a small code base, but could get you in trouble in a large legacy system!